tx-source/site_core.php

```diff
@@ -1610,6 +1610,7 @@
 $Definition['Use My Current Password'] = 'Use My Current Password';
 $Definition['User'] = 'User';
 $Definition['User "%s" does not exist.'] = 'User "%s" does not exist.';
+$Definition['UserMatchNeedsPassword'] = 'You are trying to connect with a username that is already assigned to a user. If this is you, please enter the password you used when you created this account on this forum.';
```

Comment:
I would change the wording of the translation. Here's my suggestion:

`$Definition['UserMatchNeedsPassword'] = 'You are trying to connect with a username that is already assigned to a user on this forum. If this is your account, please enter the account password.'`

Reply:
Agreed & done.